WasatchNET/EEPROM.cs

```diff
@@ -1370,6 +1425,316 @@ public virtual bool read()
                 }
             }
 
+            try

```

Comment:
As with Driver.openAllSpectrometers(), I think all the new copy-pasted code you're adding is going to be a bit of a maintenance nightmare.  Is there a reason you can't call a common function for these things?

Reply:
This was all completely intentional. I don't say this to be combative, just to say this was considered and not just lazily thrown together. Some of these functions are total copies, other are different in important ways. I agree that the current form would be a problem for maintenance.

The reason things are like this is that I was trying to avoid the async to sync "magic" wherever possible until we can confirm that it will not cause problems for LabView. If we can confirm it won't I can make all the sync functions just call the async versions, and if we confirm it _will_ cause issues then I was going to remove the magic from all synchronous fields where it is currently being used.

I know we don't usually do forks but maybe we should fork the full async out for now, or maybe I'm just being too cautious, I dunno